### ISSUE TITLE
annual to daily

### ISSUE THREAD

**Issue by AZFARHAD24511** [cynkra/tempdisagg#53]
is it possible to implement for annual as low frequency on daily time series and is there any limitations? when i apply it i face with some error as follow whereas i have no missing values

[time]: 'date' [value]: 'Y40' 
Error in td(df_y ~ df_x, method = m, conversion = "sum", fixed.rho = 0.9,  : 
  right hand side contains NAs
> any(is.na(df_y))
[1] FALSE
> any(is.na(df_x))
[1] FALSE

**Comment by christophsax** [comment]
Yes, this should be possible. Have you seen this? https://cran.r-project.org/web/packages/tempdisagg/vignettes/hf-disagg.html

If you have `NA`s, it's probably best to interpolate them beforehand. For example, using [`tsbox:: ts_na_interpolation()`](https://docs.ropensci.org/tsbox/reference/ts_examples.html?q=interpo#ref-usage).
